core/src/main/scala/org/apache/spark/util/collection/OpenHashSet.scala

```diff
@@ -271,8 +271,12 @@ class OpenHashSet[@specialized(Long, Int) T: ClassTag](
   private def hashcode(h: Int): Int = Hashing.murmur3_32().hashInt(h).asInt()
 
   private def nextPowerOf2(n: Int): Int = {
-    val highBit = Integer.highestOneBit(n)
-    if (highBit == n) n else highBit << 1
+    if (n == 0) {
+      2
```

Comment:
True or just make the condition <= 1. It won't really matter much but is a little inconsistent.